src/topology/continuous_function/compact.lean

```diff
@@ -31,15 +31,15 @@ open bounded_continuous_function
 
 namespace continuous_map
 
-variables (α : Type*) (β : Type*) [topological_space α] [compact_space α] [normed_group β]
```

Comment:
Interestingly the docstring at the top of the file describes `β` as a metric space